WordPress/src/main/java/org/wordpress/android/sharedlogin/resolver/SharedLoginResolver.kt

```diff
@@ -93,6 +83,39 @@ class SharedLoginResolver @Inject constructor(
         )
     }
 
+    @Suppress("SwallowedException")
+    private fun runFlow(accessToken: String, sites: List<SiteModel>) {
+        val hasSites = sites.isNotEmpty()
+
+        if (hasSites) {
+            resolverUtility.copySitesWithIndexes(sites)
+        }
+
+        if (jetpackAppMigrationFlowUtils.isFlagEnabled()) {
+            dispatchUpdateAccessToken(accessToken)
+            jetpackAppMigrationFlowUtils.startJetpackMigrationFlow()
+            return
+        }
```

Comment:
@RenanLukas 👋 , I moved this here since I think in principle it should work as well in the case of self-hosted only or mixed scenarios, let me know wdyt 🙇

Reply:
After this [merge from trunk](https://github.com/wordpress-mobile/WordPress-Android/pull/17391/commits/21b702b5f1ee0b5a541875cd63f6a05769dc24da) this is no more relevant